packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -372,7 +375,7 @@ function AuthOptions({
             onPasswordLogin={onPasswordLogin}
             loginHint={loginHint}
             isLoading={isPasswordLoginLoading}
-            isLoginFlow={isForgotPasswordReturn || isLoginFlow}
+            isLoginFlow={isForgotPasswordReturn || isLoginFlow || isLogin}
```

Comment:
What is the difference between isLoginFlow and isLogin? I think it's quite confusing to understand

Reply:
Discussed this here https://github.com/dailydotdev/apps/pull/2665#discussion_r1499212196

> `isLoginFlow` is controlled via the `loginState` in `AuthContext`, so I would have to import it to update that state
https://github.com/dailydotdev/apps/blob/4585cc58ef9fb3c936139394ef9ba9486008dc60/packages/shared/src/components/auth/AuthModal.tsx#L72C22-L72C34

Reply:
The entire auth is in need of some overhaul to cleanup the mishmash of solutions.

Reply:
Yup agree, anyway let's leave that for another time.

Reply:
Another time™

Reply:
Yeah there's too many abuse in the whole auth.
Also noticed this morning we actually have 3x the code verification components for different placements